src/plugins/bnbsmartchain/index.tsx

```diff
@@ -6,6 +6,14 @@ import * as unchained from '@shapeshiftoss/unchained-client'
 import { getConfig } from 'config'
 import { type Plugins } from 'plugins/types'
 
+// I'm an unchained-client wrapper around `/api/v1/` endpoints
+// If you ever need to consume endpoints which are not abstracted by chain-adapters, consume me
+export const http = new unchained.bnbsmartchain.V1Api(
```

Comment:
Now exported from all EVM plugins as a convenience utility - precisely what the comment says